### PR TITLE
[INLONG-3159][Audit] Store support TubeMQ

### LINKED ISSUE #3159: [Feature][Audit] Store support TubeMQ
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by pocozh**
### Title Name: [INLONG-3159][Audit] Store support TubeMQ

Fixes #3159 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-audit/audit-store/src/test/resources/application-test.properties [hunk 1]
```diff
@@ -16,16 +16,13 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-
```
Comment (healchow) [inline inlong-audit/audit-store/src/test/resources/application-test.properties:19]: It is recommended to keep these blank lines to make the properties file more readable.

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/config/MessageQueueConfig.java [hunk 2]
```diff
@@ -47,4 +48,27 @@
     @Value("${audit.pulsar.client.concurrent.consumer.num:1}")
     private int concurrentConsumerNum = 1;
 
+    @Value("${audit.tube.masterlist}")
+    private String tubeMasterList;
```
Comment (baomingyu) [inline inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/config/MessageQueueConfig.java:52]: this is optional config, so need config default value.

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/consume/TubeConsume.java [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            this.topic = topic;
+        }
+
+        @Override
+        public void run() {
+            ConsumerResult csmResult;
+
+            // wait partition status ready
+            while (true) {
+                if (pullMessageConsumer.isPartitionsReady(5000) || pullMessageConsumer.isShutdown()) {
```
Comment (baomingyu) [inline inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/consume/TubeConsume.java:102]: add this stat log.

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/consume/TubeConsume.java [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+                                if (StringUtils.equals(message.getTopic(), topic)) {
+                                    String body = new String(message.getData(), StandardCharsets.UTF_8);
+                                    handleMessage(body);
+                                }
+                            }
+                        }
+                        pullMessageConsumer.confirmConsume(csmResult.getConfirmContext(), true);
+                    } else {
+                        //TODO improve error handle
+                        LOG.error("receive messages errorCode is {}, error meddage is {}", csmResult.getErrCode(),
```
Comment (baomingyu) [inline inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/consume/TubeConsume.java:128]: Does this branch need ack?

**Review by baomingyu — CHANGES_REQUESTED** [review]

File: inlong-audit/audit-store/src/test/resources/application-test.properties (hunk 1 above)
Comment (pocozh) [reply inlong-audit/audit-store/src/test/resources/application-test.properties:19]: ok

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/consume/TubeConsume.java (hunk 3 above)
Comment (pocozh) [reply inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/consume/TubeConsume.java:102]: donw

File: inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/consume/TubeConsume.java (hunk 4 above)
Comment (pocozh) [reply inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/consume/TubeConsume.java:128]: no need

**Review by baomingyu — APPROVED** [review]
+1

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**